modules/op-results-search-results.adoc

```diff
@@ -0,0 +1,35 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-results-for-openshift-pipelines-observability.adoc
+
+:_content-type: PROCEDURE
+[id="results-search-results_{context}"]
+= Searching for results
+
+[role="_abstract"]
+You can search for results using Common Expression Language (CEL) queries. For example, you can find results for pipeline runs that did not succeed. However, most of the relevant information is not contained in result objects; to search by the names, completion times, and other data, search for records.
+
+.Prerequisites
+
+* You installed {tekton-results}.
+* You installed the `opc` utility and prepared its environment to quety {tekton-results}.
+
+.Procedure
+
+* To search for results using a CEL query, run the following command:
++
+[source, terminal]
+----
+$ opc results list --addr ${RESULTS_API} --filter=<CEL query>
```

Comment:
```
$ opc results list --filter="!(summary.status == SUCCESS)"
Error: accepts 1 arg(s), received 0
```
(I have addr in my config file)

Reply:
@avinal @khrm could ypu pleas ehlp me out on this command? I have no idea how to craft one here?

Reply:
I think SUCCESS isn't string but int.

Reply:
And what should the command look like then? Of you mean it is actually string and there should be single quotes around SUCCESS?

Reply:
You forgot to provide parent,

```suggestion
$ opc results list <namespace> --addr ${RESULTS_API} --filter=<CEL query>
```